pages/common/latexdiff.md

```diff
@@ -0,0 +1,16 @@
+# latexdiff
+
+> Determine differences between two LaTeX files.
+> More information: <https://ctan.org/pkg/latexdiff>.
+
+- Determine changes between different versions of a LaTeX file. The resulting LaTeX file can be compiled and shows differences underlined:
+
+`latexdiff {{old.tex}} {{new.tex}} > {{diff.tex}}`
+
+- Determine changes between different versions of a LaTeX file. Differences are highlighted in boldface:
+
+`latexdiff --type=BOLD {{old.tex}} {{new.tex}} > {{diff.tex}}`
+
+- Determine changes between different versions of a LaTeX file, show minor changes in equations and and both added and deleted graphics:
```

Comment:
```suggestion
- Determine changes between different versions of a LaTeX file, and display minor changes in equations with both added and deleted graphics:
```
I think this would be better.

Reply:
okay :)